src/Microsoft.Health.Dicom.Web/appsettings.Development.json

```diff
@@ -1,17 +1,23 @@
 {
+  "BlobStore": {
+    "ConnectionString": "UseDevelopmentStorage=true"
+  },
+  "DicomFunctions": {
+    "ConnectionName": "BlobStore:ConnectionString"
```

Comment:
Should `ConnectionName` be in appsettings.json?

Reply:
It is pretty specific to running locally, so I'm reluctant to put it outside of the development settings

Reply:
But I have changed the format of the config to better illustrate it isn't necessarily the same as blob storage